tools/influx_data_manager/influx_data_manager.xml

```diff
@@ -0,0 +1,82 @@
+<tool id="influx_data_manager" name="Influx Data Manager: Handling influx_si data inputs in Galaxy workflows" version="@TOOL_VERSION@+galaxy0" profile="21.09">
+    <macros>
+        <token name='@TOOL_VERSION@'>1.0.0</token>
+    </macros>
+    <requirements>
+        <requirement type='package' version='@TOOL_VERSION@'>influx-si-data-manager</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    influx_data_manager
+    --physiofit $in_physiofit 
+    --isocor $in_isocor 
+    --linp $in_linp 
+    --netw $in_netw 
+    --mmet $in_mmet 
+    --cnstr $in_cnstr 
+    --tvar $in_tvar 
+    --opt $in_opt 
+    --log $out_log
+    #if $verbose == 'true':
+        -v
+    #end if 
+    2> $err_out
+    ]]></command>
+    <inputs>
+        <param type='data' format='tabular, txt' name='in_netw' label='netw' help='File containing network for experiments (.netw file)' optional='false'/>
+        <param type='data' format='tabular, txt' name='in_isocor' label='miso' help='Output of isocor which corresponds to the .miso input' optional='false'/>
+        <param type='data' format='tabular, txt' name='in_tvar' label='tvar' help='File containing types of variables for experiments (.tvar file)' optional='false'/>
+        <param type='data' format='tabular, txt' name='in_physiofit' label='mflux' help='Output of physiofit which corresponds to the .mflux input' optional='true'/>
+        <param type='data' format='tabular, txt' name='in_linp' label='linp' help='File contianing label inputs for experiments (.linp file)' optional='true'/>
+        <param type='data' format='tabular, txt' name='in_mmet' label='mmet' help='File containing specie concentrations for experiments (.mmet file)' optional='true'/>
+        <param type='data' format='tabular, txt' name='in_cnstr' label='cnstr' help='File containing constraints onf fluxes and specie concentrations for experiments (.constr file)' optional='true'/>
+        <param type='data' format='tabular, txt' name='in_opt' label='opt' help='File containing options for influx_si (.opt  file)' optional='true'/>
+        <param name='verbose' type='select' label='Debug mode' help='Select "Yes" to have debug logs be printed out'>
```

Comment:
You can use `truevalue=-v` and `falsevalue=""`

Reply:
In a review for another tool I was told that it is better to use select instead of a boolean. Is this not the case anymore?

Reply:
I think that only applied for booleans in a conditional, see here for more explanation. https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#dynamic-options

Your boolean here is not changing a conditional behavior, so this is fine and it should be a normal boolean parameter imho.